src/templates/admin/journal/article_log.html

```diff
@@ -5,6 +5,11 @@
 {% block title-section %}Article {{ article.pk }} Log{% endblock %}
 {% block title-sub %}{{ article.safe_title }}{% endblock %}
 
+{% block breadcrumbs %}
+    {{ block.super }}
+    <li> {{article.safe_title}} Log</li>
```

Comment:
What if this breadcrumb were Dashboard / Logs / [Article title]? As it is it might read like "log" is a part of the article title.

Reply:
good point!

It can't be Dashboard / Logs / [Article title] because that messes with the expected breadcrumb format where each  part between the / is expected to be a link to a page.  And making 'logs' not a link would cause confusion - but I do agree it could cause confusion in current form.  I'll need to rethink and try a few alternatives.

Reply:
This is a tricky one.  There is no good solution in current form - putting Log in as a previous breadcrumb when there is no Log page is a problem, likewise putting Log in as current could confuse the article title. 

But a full review of breadcrumbs and update is already being planned as part of a11y work.  So we need a 'good enough for now' solution that causes the least confusion.  

I am therefore going to put in a "Logs" breadcrumb that is not a hyperlink.  This means the article title is non distorted, and its clear there is no such page as it is not clickable so at least users won't be hunting for the page.  Its inelegant as this implies there should be a logs page in the structure when there isn't one - but seems the best solution for now given the pending breadcrumb review.

Reply:
<img width="599" alt="image" src="https://github.com/BirkbeckCTP/janeway/assets/5330770/b6720b85-119a-4454-a9e5-b69b6f8cc397">